src/pages/Models/Details/ModelDetails.js

```diff
@@ -168,6 +168,7 @@ const ModelDetails = () => {
               filterToggle={viewFilterToggle}
             />
             <Filter
```

Comment:
I'm not sure this approach is any better than just removing this component here, even if temporary.